lib/rules/number-no-partial-percent/index.js

```diff
@@ -30,8 +30,12 @@ const rule = (primary) => {
 
 			if (!value.includes('%')) return;
 
-			// Match that start of color functions and percent values
-			const matches = value.matchAll(/(?:rgba?|color)\s*\(|[\d.]+%/g);
+			// Match the start of color functions,
+			// functions where their content should be ignored and percent value can be found inside of them (such as `url("data:image/svg+xml,203%22")`),
```

Comment:
question

Should it trigger `transform: translateX(50%);`?

Reply:
`transform: translateX(50%);` will trigger a linting error from this rule; there are also a few tests that check such cases in this PR